doc/manuals/simulation-configuration.md

```diff
@@ -1,9 +1,111 @@
+
+```json
+AUTH_PASS="pass"
+AUTH_USER="user"
+AUTH_PORT=5001
+AUTH_HOST="localhost"
+
+DOMAIN_SERVICE="service"
+DOMAIN_SUBSERVICE="/subservice"
+
+CB_HOST="localhost"
+CB_PORT=1026
+
+IOTA_ULTRALIGHT_API="1ifhm6o0kp4ew7fi377mpyc3c"
+IOTA_ULTRALIGHT_HTTP_HOST="localhost"
+IOTA_ULTRALIGHT_HTTP_PORT=8085
+IOTA_ULTRALIGHT_MQTT_HOST="localhost"
+IOTA_ULTRALIGHT_MQTT_PORT=1833
+IOTA_ULTRALIGHT_MQTT_USER="mqttUser"
+IOTA_ULTRALIGHT_MQTT_PASS="mqttPassword"
+
+IOTA_JSON_API="83ut64ib3gzs6km6izubjyenu"
+IOTA_JSON_HTTP_HOST="localhost"
+IOTA_JSON_HTTP_PORT=8185
+IOTA_JSON_MQTT_HOST="localhost"
+IOTA_JSON_MQTT_PORT=1883
+IOTA_JSON_MQTT_USER="mqttUser"
+IOTA_JSON_MQTT_PASS="mqttPassword"
+```
+
+To use a environment variable on the configuration file, it's needed to be setted around `${}` having as result `${VARIABLE}`.
+
+An example simulation configuration file using environment variables is shown next to give you a glimpse of its shape:
+
+```json
+{
+  "domain": {
+    "service": "${DOMAIN_SERVICE}",
+    "subservice": "${DOMAIN_SUBSERVICE}"
+  },
+  "contextBroker": {
+    "protocol": "http",
+    "host": "${CB_HOST}",
+    "port": "${CB_PORT}",
+    "ngsiVersion": "2.0"
+  },
+  "authentication": {
+    "provider": "keystone",
+    "protocol": "http",
+    "host": "${AUTH_HOST}",
+    "port": "${AUTH_PORT}",
+    "user": "${AUTH_USER}",
+    "password": "${AUTH_PASS}",
+    "retry": {
+      "times": 10,
+      "interval": 1000
+    }
+  },
+  "iota": {
+    "ultralight": {
+      "api_key": "${IOTA_ULTRALIGHT_API}",
+      "http": {
+        "protocol": "http",
+        "host": "${IOTA_ULTRALIGHT_HTTP_HOST}",
+        "port": "${IOTA_ULTRALIGHT_HTTP_PORT}"
+      },
+      "mqtt": {
+        "protocol": "mqtt",
+        "host": "${IOTA_ULTRALIGHT_MQTT_HOST}",
+        "port": "${IOTA_ULTRALIGHT_MQTT_PORT}",
+        "user": "${IOTA_ULTRALIGHT_MQTT_USER}",
+        "password": "${IOTA_ULTRALIGHT_MQTT_PASS}"
+      }
+    },
+    "json": {
+      "api_key": "${IOTA_JSON_API",
+      "http": {
+        "protocol": "http",
+        "host": "${IOTA_JSON_HTTP_HOST}",
+        "port": "${IOTA_JSON_HTTP_PORT}"
+      },
+      "mqtt": {
+        "protocol": "mqtt",
+        "host": "${IOTA_JSON_MQTT_HOST}",
+        "port": "${IOTA_JSON_MQTT_PORT}",
+        "user": "${IOTA_JSON_MQTT_USER}",
+        "password": "${IOTA_JSON_MQTT_PASS}"
+      }
+    }
+  }
+}
+```
+
+## Simulation configuration file
 
 As mentioned in the description of the FIWARE Device Simulator command line tool, a simulation is described by a simulation configuration file (which is passed to the command line tool using the `-c` option).
 
 The simulation configuration file is a JSON-formatted text file detailing the characteristics of the simulation to run.
 
+As it is mentioned before, this simulation configuration file can use environment variables, but, this can be overridden and be setted with different configuration if it is necessary.
```

Comment:
I don't understand what you mean by "this can be overridden and be setted with different configuration if it is necessary"...

Maybe we can clarify at skype.